<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WelcomeDialog</class>
 <widget class="QDialog" name="WelcomeDialog">
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="labelLogo">
     <property name="text"><string>CLUB MANAGER</string></property>
     <property name="alignment"><set>Qt::AlignCenter</set></property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelDesc">
     <property name="text">
      <string>Bienvenue ! Choisissez une base existante ou créez-en une nouvelle.
Accédez au tutoriel ou à la documentation à tout moment.</string>
     </property>
     <property name="alignment"><set>Qt::AlignCenter</set></property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0"><widget class="QLabel" name="labelDBPath"><property name="text"><string>Base de données :</string></property></widget></item>
     <item row="0" column="1"><widget class="QLineEdit" name="editDBPath"/></item>
     <item row="1" column="0"><widget class="QPushButton" name="buttonOpenDB"><property name="text"><string>Ouvrir une base existante</string></property></widget></item>
     <item row="1" column="1"><widget class="QPushButton" name="buttonNewDB"><property name="text"><string>Créer une nouvelle base</string></property></widget></item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item><widget class="QPushButton" name="buttonTutorial"><property name="text"><string>Tutoriel interactif</string></property></widget></item>
     <item><widget class="QPushButton" name="buttonDoc"><property name="text"><string>Documentation</string></property></widget></item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation"><enum>Qt::Horizontal</enum></property>
     <property name="standardButtons"><set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set></property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>